projects/ad7616-sdz/builds.json

```diff
@@ -0,0 +1,8 @@
+{
+      "xilinx": {
+        "demo": {
+              "flags" : "",
+              "hardware" : ["ad7616_sdz_zed", "ad7616_sdz_zc706"]
+        }
+      }
+}
```

Comment:
I would fix this warning for all the files.

Reply:
done